lib/git-fastclone.rb

```diff
@@ -217,7 +217,8 @@ def clone(url, rev, src_dir, config)
       with_git_mirror(url) do |mirror, attempt_number|
         clear_clone_dest_if_needed(attempt_number, clone_dest)
 
-        clone_command = '--quiet --reference :mirror :url :path'
+        clone_command = '--reference :mirror :url :path'
+        clone_command += verbose ? '--verbose' : '--quiet'
```

Comment:
Is `--verbose` passed to fastclone from anywhere that we know of today? Are we planning on adding it somewhere or using it as a one-off?